catalog/core/catalog-core-resourcesizeplugin/src/main/java/ddf/catalog/plugin/resourcesize/metacard/MetacardResourceSizePlugin.java

```diff
@@ -65,7 +66,7 @@ public QueryResponse process(QueryResponse input)
                     } finally {
                         Thread.currentThread().setContextClassLoader(tccl);
                     }
-                } catch (Exception e) {
```

Comment:
It seems that the only exception that can be thrown here is the CacheException.  Before CacheException was removed, it was the only exception being caught.  https://github.com/codice/ddf-catalog/commit/e90db6d5d82820a7dfd9d04b0b305c475fb20e6c